zonemaster/zonemaster-gui
# don't overwrite the config file on update

## Purpose

When copying the new files after an update the configuration could be overwritten as is it present in the source.

## Changes

This PR moves the configuration out of the source and provides instruction to create the file.

## How to test this PR

* Build the GUI and make an archive from the dist directory
* Unarchive the archive to a temporary directory
* Create and edit the configuration
* Unarchive the archive to the same directory
* The configuration should remain untouched
